entries/p/paypal.com.json

```diff
@@ -4,7 +4,8 @@
       "paypal-search.com"
     ],
     "mfa": "allowed",
-    "documentation": "https://www.paypal.com/us/smarthelp/article/faq4057",
+    "notes": "An eligible Apple or Android device is required to create a passkey.",
```

Comment:
I don't think this is needed, even though it's stated in the documentation. Do you know if passkey creation is actually limited to those devices or if it's more guidance than fact?

Reply:
I cannot check myself, but judging by Shannon Morse's guide video on the topic, it looks like it is indeed a restriction: https://www.youtube.com/watch?v=3jEm5WyN4EQ

Reply:
I just checked, and sadly, it is true. There are no limitations that prevent them from accepting other browsers/platforms. In fact, I just set it up using Firefox and a user-agent switcher.